docs/jade-getting-started.md

```diff
@@ -301,26 +272,8 @@ export JADE_USER_EMAIL=<EMAIL_YOU_CREATED_FOR_DEVELOPMENT>
 # Integration test setting
 export IT_JADE_API_URL=http://localhost:8080
 
-# This file will be populated when you run ./render-configs.sh
-export GOOGLE_APPLICATION_CREDENTIALS=/tmp/jade-dev-account.json
-export GOOGLE_SA_CERT=/tmp/jade-dev-account.pem
-
-# Setting for credentials to test on Azure - these files are populated in the render-configs.sh script
-# Defaults to dev; You can switch to integration by running `./render-config.sh -a integration`
-export AZURE_SYNAPSE_WORKSPACENAME=$(cat /tmp/azure-synapse-workspacename.txt)
-export AZURE_CREDENTIALS_HOMETENANTID=$(cat /tmp/jade-dev-tenant-id.key)
-export AZURE_CREDENTIALS_APPLICATIONID=$(cat /tmp/jade-dev-client-id.key)
-export AZURE_CREDENTIALS_SECRET=$(cat /tmp/jade-dev-azure.key)
-export AZURE_SYNAPSE_SQLADMINUSER=$(cat /tmp/jade-dev-synapse-admin-user.key)
-export AZURE_SYNAPSE_SQLADMINPASSWORD=$(cat /tmp/jade-dev-synapse-admin-password.key)
-export AZURE_SYNAPSE_ENCRIPTIONKEY=$(cat /tmp/jade-dev-synapse-encryption-key.key)
 export AZURE_SYNAPSE_INITIALIZE=false
 
-# RBS
-# Defaults to RBS tools; you can switch to dev by running `./render-configs.sh -r dev`
-export RBS_POOLID=$(cat /tmp/rbs-pool-id.txt)
-export RBS_INSTANCEURL=$(cat /tmp/rbs-instance-url.txt)
-
```

Comment:
are the deleted env variables now automatically populated by running render configs?

Reply:
Yes